README.md

```diff
@@ -310,61 +310,61 @@ This means the following information about modules have been moved:
 - Module is runtime module in chunk -> ChunkGraph
 - Chunk runtime requirements -> ChunkGraph
 
-webpack used disconnect modules from the graph when restored from cache. This is no longer necessary. Module store no info about the graph and can technically used in multiple graphs. This makes caching easier.
+webpack used to disconnect modules from the graph when restored from cache. This is no longer necessary. A Module stores no info about the graph and can technically used in multiple graphs. This makes caching easier.
 
-There is a compat-layer for most of these changes which prints a deprecation warning when used.
+There is a compat-layer for most of these changes, which prints a deprecation warning when used.
 
 MIGRATION: Use the new APIs on ModuleGraph and ChunkGraph
 
-## Init fragments
+## Initialize Fragments
```

Comment:
```suggestion
## Init Fragments
```

Better keep `Init Fragments` as name as they are also called this way internally and multiple names could be confusing

Reply:
ohh right, didn't realize that. Thanks.